app/src/main/java/com/project/gamedb/data/model/ResultGames.kt

```diff
@@ -0,0 +1,25 @@
+package com.project.gamedb.data.model
+
+import com.project.gamedb.ultis.map
+import org.json.JSONObject
+
+data class ResultGames(
+        val count: Long,
+        val next: String,
+        val previous: String,
+        val results: List<Games>
```

Comment:
Mỗi lần lùi lề vào 4 space thôi anh 
```suggestion
    val count: Long,
    val next: String,
        val previous: String,
        val results: List<Games>
```